x/emissions/types/keys.go

```diff
@@ -58,9 +58,9 @@ var (
 	PreviousReputerRewardFractionKey                  = collections.NewPrefix(43)
 	PreviousInferenceRewardFractionKey                = collections.NewPrefix(44)
 	PreviousForecastRewardFractionKey                 = collections.NewPrefix(45)
-	LatestInfererScoresByWorkerKey                    = collections.NewPrefix(46)
-	LatestForecasterScoresByWorkerKey                 = collections.NewPrefix(47)
-	LatestReputerScoresByReputerKey                   = collections.NewPrefix(48)
+	InfererScoreEmasKey                               = collections.NewPrefix(46)
```

Comment:
@kpeluso can we rename here?

Reply:
I think so cuz the prefix is the same, just changing the variable, but I'm not 100%...

Reply:
If this was just a renaming of variables, but they are still the same thing, then I think it is ok.

Reply:
yeah underlying, persisted values are untouched